test/app_signals/resources/metrics/server_consumer.json

```diff
@@ -261,17 +261,17 @@
                     ],
                     "startTimeUnixNano": START_TIME,
                     "timeUnixNano": START_TIME,
-                    "sum": 1,
-                    "min": 1,
-                    "max": 1
+                    "sum": 0,
```

Comment:
Why are we changing these just curious

Reply:
This is to reflect the default app signals metrics